test/fixtures/convert/decorator/mobx/flow.js

```diff
@@ -0,0 +1,15 @@
+import { observable, computed, action } from "mobx";
+export class Store {
+  @observable
+  markersMapping: Map<number, MarkerStore> = new Map();
+
+  @computed
+  get filePath(): ?string {
+    return "/file";
+  }
+
+  @action
+  newMarkerStore(editorId: number) {
+    return 1;
+  }
+}
```

Comment:
Thanks for simplifying these examples.  The could be simpler since the code doesn't have to be semantically valid (in this case the getter doesn't return a value, e.g.
```
class MyClass {
   @property
   num: number = 5;

   @computed
   get foo(): string {}

   @action
   doSomething() {}
}
```
This is fine for now.  No action req'd.